LLPlot.py

```diff
@@ -0,0 +1,176 @@
+from PIL import Image, ImageDraw, ImageOps
+import math
+
+
+class Plot:
+    def __init__(self):
+        self.background = (168, 168, 168)
+        self.imageName = 'image1'
+        self.imageType = '.jpg'
+        self.imageSize = (1920, 1080)
+        self.g0Color = (256, 0, 0)
+        self.g1Color = (0, 256, 0)
+        self.g2Color = (0, 256, 0)
+        self.g3Color = (0, 256, 0)
+        self.lineThickness = 2
+
+        self.__min_x = 500000
+        self.__min_y = 500000
+        self.__max_x = -500000
+        self.__max_y = -500000
+
+    def set_background_color(self, color):
```

Comment:
is color the america english and colour the correct english? :)

Reply:
Added to commit 343fe27